swagger.yaml

```diff
@@ -4137,6 +4138,48 @@ definitions:
         type: string
         description: The device path.
 
+  domains_resolve_request:
```

Comment:
```suggestion
  domain_resolution_request:
```

Reply:
(For all))

Reply:
But then you can't differentiate between the inner object for a single domain and the outer object for multiple domains